docs/docs/build/bob-sdk/gateway.md

```diff
@@ -2,164 +2,135 @@
 sidebar_position: 1
 ---
 
-# BOB Gateway Integration Guide
+# BOB Gateway
 
-## What is BOB Gateway?
+## Overview
 
-[BOB Gateway](https://docs.gobob.xyz/docs/learn/guides/bitcoin-bridge/) is a trustless, P2P bridge that enables users to bridge their BTC from Bitcoin mainnet to BOB.
+[BOB Gateway](https://docs.gobob.xyz/docs/learn/guides/bitcoin-bridge/) is a Bitcoin intent bridge that unlocks Bitcoin liquidity by reducing the number of steps to onboard users, saving time and money. Users can go from **BTC** on Bitcoin to **staked BTC LSTs** with a single Bitcoin transaction.
 
-You can use our SDK to add this UX directly into your app, bringing your app's functionality (lending, swapping, restaking, etc.) directly to Bitcoin hodlers.
+Our SDK makes it possible for you to bring this UX directly into your app.
 
-## How Does it Work?
+## How Gateway Works
 
 1. Liquidity providers (LPs) temporarily lock wrapped Bitcoin (WBTC or tBTC) in escrow smart contracts on BOB.
-2. A user submits a transaction to the Bitcoin network that sends BTC to the liquidity provider's Bitcoin address and publishes the user's EVM wallet address in the `OP_RETURN` of the transaction.
-3. BOB's relayer server calls an on-chain [Light Client](../examples/btc-swap/index.mdx) to trustlessly verify the Bitcoin transaction and transfer the LP's wrapped Bitcoin to the user's EVM address, including a "gratuity" amount of ETH for transaction fees on BOB.
+1. A user makes a request for wrapped or staked Bitcoin (e.g. WBTC, tBTC, or a Bitcoin LST/LRT).
+1. The user sends BTC to the liquidity provider's Bitcoin address. A hash of the user's order is included in the `OP_RETURN` of the transaction.
+1. Gateway finalizes the transaction. After trustlessly verifying the user's Bitcoin transaction with an on-chain [Light Client](../examples/btc-swap/index.mdx), Gateway sends the LP's wrapped Bitcoin to the user's EVM address. If the user requested a Bitcoin LST/LRT, that token is minted using the LP's wrapped Bitcoin before it is sent to the user.
 
-This SDK makes it possible to do steps 2 and 3 in your application's front end.
+This SDK makes it possible to do steps 2, 3, and 4 in your application's front end.
 
 ## Step-by-Step Integration Guide
 
-Follow these steps to add BOB Gateway's functionality to your application.
+This is an example implementation of our SDK. You will need to decide how you handle asking your user to sign a partially-signed Bitcoin transaction (PSBT). We recommend using our [sats-wagmi](./sats-wagmi.md) package to connect to your users' wallets.
 
 ### Install the BOB SDK
 
-Install the `@gobob/bob-sdk` package using your preferred package manager:
+Add `@gobob/bob-sdk` to your project using your preferred package manager.
 
-```sh
-# with Yarn
-$ yarn add @gobob/bob-sdk
-
-# with npm
-$ npm i @gobob/bob-sdk
-
-# with pnpm
-$ pnpm add @gobob/bob-sdk
-
-# with Bun
-$ bun add @gobob/bob-sdk
```

Comment:
Do we wanna keep this?

Reply:
The `npm2yarn` package provides a tab-switcher between package managers, so we still have multiple options. I opted for that to strive for visual conciseness, but I'm happy to revert if you prefer.